### DISCUSSION TITLE
Are there any other examples?

### DISCUSSION THREAD

**Discussion by Allin2000** [ag2ai/faststream#1549] [General]
Cryptocurrency analysis with FastStream:This example requires an openai key.

**Comment by kumaranvpl** [comment]
Hello @Allin2000,

Checkout our quick start page - https://faststream.airt.ai/latest/getting-started/. It has a minimal example which does not needs any keys. 

Also checkout the other sections of tutorial page such as publishing - https://faststream.airt.ai/latest/getting-started/publishing/ in order to learn more about the different capabilities of FastStream
